fvm/src/machine/engine.rs

```diff
@@ -101,15 +98,19 @@ impl From<wasmtime::Engine> for Engine {
 
 struct Cache<K> {
     linker: wasmtime::Linker<InvocationData<K>>,
-    instances: HashMap<Cid, wasmtime::InstancePre<InvocationData<K>>>,
```

Comment:
This will add a ~70-200k gas overhead. But, well, we _can't_ feasibly cache instances in M2 anyways unless we find some way to persist said cache.